manager/integration/tests/test_ha.py

```diff
@@ -279,6 +293,100 @@ def ha_salvage_test(client, core_api, # NOQA
 
     cleanup_volume(client, volume)
 
+    # Setting: Enabled auto salvage.
+    # Case 3: Revision counter disabled.
+
+    auto_salvage_setting = client.by_id_setting(SETTING_AUTO_SALVAGE)
+    setting = client.update(auto_salvage_setting, value="true")
+    assert setting.name == SETTING_AUTO_SALVAGE
+    assert setting.value == "true"
+
+    disable_revision_counter_setting = \
+        client.by_id_setting("disable-revision-counter")
+    setting = client.update(disable_revision_counter_setting, value="true")
+    assert setting.name == "disable-revision-counter"
+    assert setting.value == "true"
+
+    volume = create_and_check_volume(client, volume_name, 3,
+                                     backing_image=backing_image)
+
+    host_id = get_self_host_id()
+    volume = volume.attach(hostId=host_id)
+    volume = common.wait_for_volume_healthy(client, volume_name)
+
+    assert len(volume.replicas) == 3
+    orig_replica_names = []
+    for replica in volume.replicas:
+        orig_replica_names.append(replica.name)
+
+    data = write_volume_random_data(volume)
+
+    crash_replica_processes(client, core_api, volume_name)
+
+    volume = common.wait_for_volume_faulted(client, volume_name)                
+    assert len(volume.replicas) == 3
+
+    volume = common.wait_for_volume_healthy(client, volume_name)
+    assert len(volume.replicas) == 5
```

Comment:
@shuo-wu according to tests steps, replica count should be 5, but I only get 3
is that expected.

Reply:
3-replica is what we expected now. Because this test skeleton is written before the failed replica reusage feature, the 2 failed replicas mentioned here will be reused automatically now.

You can modify the test skeleton to:
```
7. Verify all 3 replicas are reused successfully.
```

Reply:
Fixed, Thank you.